src/schema/rules/tabular_metadata.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
@effigies  we already have

```
❯ grep -A12 -E '^(scans|sessions)' src/schema/rules/files/common/tables.yaml
scans:
  level: optional
  suffixes:
    - scans
  extensions:
    - .tsv
    - .json
  entities:
    subject: required
    session: optional # session is required if session is present in the dataset.
sessions: # This file may only exist if session is present in the dataset.
  level: optional
  suffixes:
    - sessions
  extensions:
    - .tsv
    - .json
  entities:
    subject: required
```

seemingly with identical content (just dict not list).

How this file is used "in addition"?

Reply:
Sorry for the noise @effigies ! my bad -- how the hack I got to review this ? ;)    codecov woke up 2 years later and I thought that this PR got just merged, so I got all confused